lib_xcore_math/src/bfp/bfp_alloc.c

```diff
@@ -11,7 +11,7 @@
 bfp_s32_t bfp_s32_alloc(
     const unsigned length)
 {
-#if (DEBUG)
+#if (defined DEBUG && DEBUG > 0)
```

Comment:
Is this config even being defined?

Reply:
I thought that it can be given via CLI. It is not defined within the code.